diesel/src/expression_methods/global_expression_methods.rs

```diff
@@ -3,6 +3,7 @@ use expression::array_comparison::{AsInExpression, In, NotIn};
 use expression::operators::*;
 use types::SingleValue;
 
+/// Methods present on all expressions, except tuples
 pub trait ExpressionMethods: Expression + Sized {
```

Comment:
We should remove the `unwrap` statements and raw sql strings in the example code of:
* `eq_any` 
* `ne_any`

Reply:
I didn't think it was worth the churn since those lines aren't rendered.